lib/ukintctlr/ukintctlr.c

```diff
@@ -40,6 +41,7 @@
 #include <uk/arch/ctx.h>
 #endif
 
+#define MAX_IRQ				UK_INTCTLR_MAX_IRQ
```

Comment:
Good point, will do 👍🏼

Reply:
Actually hold on. If we do that we will also need to do the same for the definitions of the IRQ allocation limits. Do we really want to add these checks? Because: If the driver does not define `limits.h` the build will fail. If the driver defines `limits.h` without these values, the build will fail again. So do we get any benefit from these checks after all? 🤔

Reply:
> benefit from these checks after all

Shouldn't the driver be enforced to define these? If we go without `#error`, I think we should at least have some default values instead? :thinking:

Reply:
Sure, I don't have a strong opinion, either way is fine with me. I'll add a few check then, as defining defaults for alloca(ta)ble interrupt numbers sounds a bit risky to me 👍🏼